src/constants/ChainConstants.ts

```diff
@@ -56,8 +56,8 @@ export class ERCChainConstants extends ChainConstants {
 }
 
 function getChainId() {
-  if (!window.tronWeb) return undefined;
-  const host = window.tronWeb.fullNode.host;
+  if (!window.tronWeb?.fullNode?.host) return undefined;
+  const host = window.tronWeb.fullNode?.host;
```

Comment:
if window is undefined it will throw error here.
```suggestion
  const host = window.tronWeb?.fullNode?.host;
```

Reply:
Browser throwing error for below code if I don't check for fullNode.host

```
if (host.includes('api.trongrid')) { // Throw error here
    return SupportedChainId.TRON_MAINNET;
  } else if (host.includes('api.shasta')) { // Throw error here
    return SupportedChainId.TRON_SHASTA_TESTNET;
  } else if (host.includes('api.nile')) { // Throw error here
    return SupportedChainId.TRON_NILE_TESTNET;
  }
```